### PR TITLE
Userform

### PR DESCRIPTION
**Pull request by edignot**
## Pull request checklist
Please check if your PR fulfills the following requirements:
- [ ] Lint (`npm run lint`) has passed locally and any fixes were made for failures

## Pull request type
 
 
Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Testing
- [ ] Other (please describe): 

## What is the current behavior?

-  

## What is the new behavior?

- User image upload functionality reused 
- React Icons added to user info card
- Minor layout bugs fixed ( align, flex .. )

## Does this introduce a breaking change?
- [ ] Yes
- [x] No

## Other information

- Need to save image url to the database ( can ben sent with the same user update form )

### CODE REVIEW

File: code/web/src/modules/user/UserInfo.js [hunk 1]
```diff
@@ -1,249 +1,304 @@
...
-        const imageData = new FormData();
-        imageData.append('image', e.target.files[0]);
-        // file imageData will be send to the database
-        console.log('IMAGE DATA', imageData);
-        // temporally imgUrl
-        this.setState({
-            imageUrl: URL.createObjectURL(e.target.files[0]),
-        });
-    };
+  imageSelectedHandler = (e) => {
```
Comment (KyleWong2510) [inline code/web/src/modules/user/UserInfo.js:42]: Nice touch!

File: code/web/src/modules/user/UserInfo.js [hunk 2]
```diff
@@ -1,249 +1,304 @@
...
+
+          this.setState({
+            ...user,
+          })
+        } else {
+          this.props.messageShow('Please try again.')
+        }
+      })
+      .catch((error) => {
+        this.props.messageShow('There was some error. Please try again.')
```
Comment (KyleWong2510) [inline code/web/src/modules/user/UserInfo.js:65]: Loving the confirmations and error handling!  Very nice UX

File: code/web/src/modules/user/UserInfo.js [hunk 3]
```diff
@@ -1,249 +1,304 @@
...
+      })
+      .then(() => {
+        window.setTimeout(() => {
```
Comment (KyleWong2510) [inline code/web/src/modules/user/UserInfo.js:68]: Usually dislike setTimeout but this is a great implementation!

File: code/web/src/modules/user/UserInfo.js [hunk 4]
```diff
@@ -1,249 +1,304 @@
...
+    )
+  }
+  renderInfo = () => {
+    return (
+      <section style={infoContainer}>
+        <section style={imageEmailNameWrapper}>
+          <div style={imageWrapper}>
+            <img
+              src={routeImage + this.state.imageUrl}
+              onError={this.addDefaultImageSrc}
```
Comment (KyleWong2510) [inline code/web/src/modules/user/UserInfo.js:156]: This onError is a smart way to handle this.

File: code/web/src/modules/user/api/actions.js [hunk 5]
```diff
@@ -54,106 +58,72 @@ export function login(userCredentials, isLoading = true) {
...
-                    type: LOGIN_RESPONSE,
-                    error: 'Please try again',
-                });
-            });
-    };
+          type: LOGIN_RESPONSE,
+          error: 'Please try again',
+        })
+      })
+  }
```
Comment (KyleWong2510) [inline code/web/src/modules/user/api/actions.js:70]: The fully working POST will be on my branch.  I will push it up once this is approved and merged.

File: code/web/src/modules/user/DeliveryInfo.js [hunk 6]
```diff
@@ -23,38 +24,49 @@ class DeliveryInfo extends React.Component {
...
-          </section>
-        }
-        <section>
-          <p>Product History</p>
-          <p>product product product</p>
-          <p>product product product</p>
-          <p>product product product</p>
-          <p>product product product</p>
+      <section style={style}>
+        {/* {this.state.isEditing ? (
```
Comment (sagemlee) [inline code/web/src/modules/user/DeliveryInfo.js:28]: Is this supposed to be all commented out?

File: code/web/src/modules/user/UserInfo.js (hunk 3 above)
Comment (sagemlee) [reply code/web/src/modules/user/UserInfo.js:68]: What does this do?

**Pull request merged**